bors.toml

```
@@ -5,3 +5,4 @@ block_labels = ["not-ready-to-merge", "WIP", "blocked-by-other-PR", "merge-confl
 delete_merged_branches = true
 update_base_for_deletes = true
 cut_body_after = "---"
+max_batch_size=16
```

Not sure it matters, but the rest of the file puts spaces here
```suggestion
max_batch_size = 16
```